public/app/features/alerting/unified/components/AlertLabelDropdown.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+
+import { SelectableValue } from '@grafana/data';
+import { Select, Field } from '@grafana/ui';
+
+export interface AlertLabelDropdownProps {
+  onChange: (newValue: SelectableValue<string>) => void;
+  options: SelectableValue[];
+  defaultValue?: SelectableValue;
+  type: 'key' | 'value';
+}
+
+const AlertLabelDropdown: FC<AlertLabelDropdownProps> = React.forwardRef(function labelPicker(
```

Comment:
Why do you need this forwardRef ? Actually, this ref is not used in any place

Reply:
I initially implemented it without the `forwardRef` but React Hook Form complains about it: 
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/6271380/196186030-4187d3d4-66e4-431d-916b-2d7367fce9f6.png">

In their docs they say to use it when integrating existing forms https://react-hook-form.com/get-started#integratinganexistingform

I'm not sure why it's needed in this case, but by adding it the warning went away. I did change it to apply it to the component as you smartly spotted!

Reply:
ah! okok, yes, then you can remove the ref param 👍